contrib/miniupnpd/files/miniupnpd

```diff
@@ -0,0 +1,6 @@
+# miniupnpd service
+
+type = process
+command = /usr/bin/miniupnpd -f /etc/miniupnpd/miniupnpd.conf
+depends-on = pre-network.target
```

Comment:
auxilitary target; you need thís and probably local.target dependencies

Reply:
got it